webpack.config.js

```diff
@@ -12,7 +12,8 @@ var wpconfig = {
     context: __dirname,
 
     entry: {
-        CourseOutline: './openedx/features/course_experience/static/course_experience/js/CourseOutline.js'
+        CourseOutline: './openedx/features/course_experience/static/course_experience/js/CourseOutline.js',
```

Comment:
The advantage of having one common Webpack config is that [once we implement CommonsChunkPlugin](https://openedx.atlassian.net/browse/LEARNER-642) Webpack can generate one shared vendor chunk for both Studio and LMS, so users of both only need to download Backbone/jQuery/React/Moment/other large libs once and we can cache it on the CDN and on users' devices as for long as we continue using those versions of those libraries. In our current RequireJS setup, a user of both Studio and LMS is downloading all those bytes twice (maybe more).